config/components/manager/controller_manager_config.yaml

```diff
@@ -41,6 +41,8 @@ integrations:
   - "kubeflow.org/pytorchjob"
   - "kubeflow.org/tfjob"
   - "kubeflow.org/xgboostjob"
+  externalFrameworks:
+  - "workload.codeflare.dev/v1beta2, Kind=AppWrapper"
```

Comment:
This should be after `- "pod"`, it should be commented, and it shouldn't have a specific type mentioned.

Maybe put `example.com/v1, Kind=Foo`